internal/commands/untrust_builder.go

```diff
@@ -28,6 +29,12 @@ func UntrustBuilder(logger logging.Logger, cfg config.Config) *cobra.Command {
 			}
 
 			if len(existingBuilders) == len(cfg.TrustedBuilders) {
+				// Use empty config to just search through suggested builders
+				if isTrustedBuilder(config.Config{}, builderName) {
+					// Attempted to untrust a suggested builder
+					return errors.Errorf("Builder %s is a suggested builder, and is trusted by default. Currently pack doesn't support making these builders untrusted", style.Symbol(builderName))
+				}
+
```

Comment:
I would add a new function `isSuggestedBuilder` to `suggested_builders.go` and call it instead of the `isTrustedBuilder` call.
Its implementation is pretty simple:
```
func isSuggestedBuilder(builder string) bool {
	for _, sugBuilder := range suggestedBuilders {
		if builder == sugBuilder.Image {
			return true
		}
	}

	return false
}
```
and it should be called by `isTrustedBuilder` in `commands.go`.

Reply:
Agreed. Having this comment "Use empty config to just search through suggested builders" points out that the interface is a little hard to comprehend.